src/app/features/balances-list/balances-list.tsx

```diff
@@ -20,11 +22,12 @@ import { PendingBrc20TransferList } from '../pending-brc-20-transfers/pending-br
 import { BitcoinFungibleTokenAssetList } from './components/bitcoin-fungible-tokens-asset-list';
 import { StacksFungibleTokenAssetList } from './components/stacks-fungible-token-asset-list';
 
-interface BalancesListProps extends StackProps {
+interface BalancesListContextState {
   address: string;
 }
 
```

Comment:
I removed `...props` from here as I couldn't see it doing anything. That could be incorrect though and I can re-add if needed

Reply:
Yeah, I don't see any props being passed, so I think it makes sense to remove it.